File: Domain/Entities/Event.cs
using EventManagement.Application.Models;

namespace EventManagement.Domain.Entities

{ 
    public class Event
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime Date { get; set; }
        public string Location { get; set; }
        public string Category { get; set; }
        public int MaxParticipants { get; set; }
        //add image 
        public byte[] ImageData { get; set; }
        
        public List<EventParticipant> EventParticipants { get; set; }
    }
}

File: Client/Client/src/models/Event.ts
export interface IEvent{
    id:number;
    
    name:string;
    description:string;
    date:Date;
    location:string;
    category:string;
    maxParticipants:number;
    imageSrc:string;
    
}

export interface IEventCreate {
    name: string;
    description: string;
    date: Date;
    location: string;
    category: string;
    maxParticipants: number;
    imageData: File;
}